frontend/src/components/filters/descriptions.js

```diff
@@ -244,5 +244,31 @@ export const USER_FILTERS = {
     type: FILTER_TYPE.INPUT,
     key: "email",
     label: "Email"
-  }, 
+  },
+}
+
+export const PROJECT_FILTERS = {
```

Comment:
Could be great to have a filter on the request_date of a project (see Experiment start_date or Sample creation_date as examples of date range filter)